src/test/porter/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: porter
+description: Porter chart for KubeSphere
+type: application
+version: 0.1.1
+appVersion: 0.1.1
+icon: https://kubesphere.io/static/logo.16097928.svg
```

Comment:
The URL reports "HTTP 404 Not Found". Do we have a stable URL for ks logo?

Reply:
I think we need to add a Porter logo into this chart, instead of kubesphere logo.

Reply:
Yes, URL for Porter is preferred. If no URL, this `icon` field can be removed. It's optional.

Reply:
@hlwanghl  Can we add the porter.svg to kubesphere/porter repo, and replace the icon URL with a GitHub address? For example:
https://raw.githubusercontent.com/FeynmanZhou/landscape/master/hosted_logos/porter.svg

Reply:
sounds good